docs/software/media.md

```diff
@@ -47,4 +43,818 @@
+    When you reach Setup your media libraries:
+
+    ```sh
+    /mnt/dietpi_userdata/Music
+    /mnt/dietpi_userdata/Video
+    ```
+
+    You can also mount Samba/NFS shares by running `dietpi-drive_manager` and selecting `Mount network drive` from the menu.
+
+=== "Transfer media files to your device"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by Emby:
+
+    - audio = `/mnt/dietpi_userdata/Music`
+    - videos = `/mnt/dietpi_userdata/Video`
+
+=== "Update"
+
+    To update the Emby Server version simply reinstall Emby Server to update it to newest version. It will preserve any existing data and settings:  
+    `dietpi-software reinstall 41`
+
+## Plex Media Server - Web interface media streaming server
+
+Plex organizes your video, music, and photo collections and streams them to all of your screens. Also known as a "Web interface media streaming server".
+
+![DietPi media server software Plex Media Server](../assets/images/dietpi-software-media-plexmediaserver.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    URL = `http://<your.IP>:32400/web`
+
+=== "First run setup"
+
+    Please follow the onscreen instructions to complete the setup wizard:
+
+    - Create your Plex account by signing up.
+    - Give your server a name (e.g.: `DietPi-Plex`)
+    - Click `library` to setup your media locations:
+
+    ```sh
+    /mnt/dietpi_userdata/Music
+    /mnt/dietpi_userdata/Video
+    ```
+
+    You can also mount Samba/NFS shares by running `dietpi-drive_manager` and selecting `Mount network drive` from the menu.
+
+=== "Transfer media files to your device"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by Plex Media Server:
+
+    - audio = `/mnt/dietpi_userdata/Music`
+    - videos = `/mnt/dietpi_userdata/Video`
+
+## Tautulli - Monitoring and tracking tool for Plex Media Server
+
+A Python-based monitoring and tracking tool for Plex Media Server.
+
+![DietPi media server software Tautulli](../assets/images/dietpi-software-media-tautulli.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    URL = `http://<your.IP>:8181`
+
+=== "Access to the log files"
+
+    `/mnt/dietpi_userdata/plexpy/logs/`
+
+## Murmur - Mumble VoIP Server
+
+Turn your device into a lightweight Mumble VoIP server.
+
+![DietPi media server software Murmur](../assets/images/dietpi-software-media-murmur.png){: style="width:100px"}
+
+=== "Murmur Server connection details"
+
+    Use the following credentials to connect clients to the Murmur server:
+
+    - URL = `http://<your.IP>:64738`
+    - Password = No password required
+    - Superuser Password = `dietpi`
+
+=== "Change Murmur Server settings"
+
+    You will need to edit the Murmur config file:
+
+    ```sh
+    nano /etc/mumble-server.ini
+    #Restart service
+    service mumble-server restart
```

Comment:
Let's do it the native systemd way: `systemctl restart mumble-server`

Reply:
Ok.

@MichaIng: Would you please check other sections with `dietpi-services restart` for a change after the merge into the master branch?  
I found two additional cases in section IceCast: lines 730 and 753